DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -58,18 +58,12 @@ public LoginResult AttemptLogin(string username, string password)
             if (shouldIncreaseFailedLoginCount)
             {
                 userService.IncrementFailedLoginCount(unverifiedAdminUser!);
+                unverifiedAdminUser!.FailedLoginCount += 1;

```

Comment:
Is this incrementing it twice? `IncrementFailedLoginCount` also looks to be incrementing the count when calling `UpdateAdminUserFailedLoginCount`